src/implementations/twig/components/page-header-standardised/__snapshots__/page-header-standardised.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`Page Header Standardised Background image - renders correctly 1`] = `
 <jest>
   <div
-    class="ecl-page-header-standardised ecl-page-header-standardised--image"
+    class="ecl-page-header-standardised ecl-page-header-standardised--image ecl-page-header-standardised--overlay-none"
```

Comment:
Yep, this is the consequence of the fact that we pass the 'none' value for the overlay parameter..